tests/Exercism.TestRunner.FSharp.IntegrationTests/Solutions/CompileError/Fake.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project file should be an F# project files (.fsproj), as the integration tests should run against F# projects. For a sample F# project file, see [this file](https://github.com/exercism/fsharp/blob/master/exercises/accumulate/AccumulateTest.fs).

Reply:
Note that this comment applies to all the F# project files within the `tests/Exercism.TestRunner.FSharp.IntegrationTests/Solutions` directory.

Reply:
Ported all projects `tests/Exercism.TestRunner.FSharp.IntegrationTests/Solutions` directory to F#